profiles/seccomp/seccomp_test.go

```diff
@@ -38,8 +38,46 @@ func TestLoadDefaultProfile(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	rs := oci.DefaultSpec()
+	rs := createSpec()
 	if _, err := LoadProfile(string(f), &rs); err != nil {
 		t.Fatal(err)
 	}
 }
+
+// createSpec() creates a minimum spec with default capabilities for testing
+func createSpec() specs.Spec {
```

Comment:
as this seems to be used only in tests why does it matter what the spec contents is? If anything then I think we could have a test that takes empty spec, sets `CAP_ADMIN` in it, and then checks if correct seccomp profile is loaded.

Reply:
Good point, I was thinking about that. Basically, I tried to keep the test the same as it was (but only setting the minimum amount of options that are looked at when generating the profile). It would be good to test if the actual conditional behaviour works yes. Let me give that a try